db/migrate/20151204143202_create_justice_gov_sk_judges.rb

```diff
@@ -0,0 +1,20 @@
+class CreateJusticeGovSkJudges < ActiveRecord::Migration
+  def change
+    create_table :justice_gov_sk_judges do |t|
+      t.string :uri, limit: 2048, null: false
+      t.text :html, null: false
+
+      t.string :meno
+      t.string :sud
+      t.string :sud_uri, limit: 2048
+      t.string :docasny_sud
+      t.string :docasny_sud_uri
```

Comment:
skontroluj vsetky `*_uri` aby mali limit 2048, nerobil by som to ako migracie ale fixol to priamo tu